apple-pie/styles/style.css

```
@@ -0,0 +1,33 @@
+
+
+.encabezado {
+    color: white;
+    font-size: 35px;
+    padding: 500px;
+    background-image: url(/apple-pie/images/apple-pie.jpg);
+    background-size: cover;
+    text-align: center
+}
+
+.texto {
+    margin: 50px 300px; 
+    text-align: justify;
```

```suggestion
    text-align: center;
```
Esto es para que el texto este en el centro, lo anterior es para toda la caja, esto para los elementos dentro de esa caja